emscripten.py

```diff
@@ -1411,7 +1411,7 @@ def asmjs_mangle(name):
         metadata['declares'].append(func_name)
     elif line.startswith('  (func '):
       parts = line.split(' ')
```

Comment:
Looks like it would. I did not know `split()` did that. That's probably more in line with a reader's expectations of how this should work, i.e. I don't think people would consider the case where there's trailing newlines after we split.